internal/flushcommon/writebuffer/stats_buffer.go

```diff
@@ -0,0 +1,49 @@
+// Licensed to the LF AI & Data foundation under one
+// or more contributor license agreements. See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership. The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package writebuffer
+
+import (
+	"github.com/milvus-io/milvus/internal/storage"
+)
+
+type statsBuffer struct {
```

Comment:
I am a little bit confused: is this `statsBuffer` designed to include BM25 stats only? or could be other types?

Reply:
yes, only bm25 stats need buffer beacause  it more complex, other stats could generated at flush.